modules/ROOT/pages/_partials/usage-runtime.adoc

```diff
@@ -0,0 +1,35 @@
+To view usage reports for Mule runtime, all of your relevant applications (in production and sandbox) must use the following Mule versions:
```

Comment:
What does "relevant" mean in this context?
Can you just say "all applications in prod and sandbox..."